### PR TITLE
Bump litd to version `v0.13.0-alpha`

### PR DESCRIPTION
**Pull request by ViktorTigerstrom**
This PR bumps the litd version to `v0.13.0-alpha`.

This PR also bumps lnd to version `v0.18.0-beta` and all sub-servers except faraday to new versions which support lnd `v0.18.0-beta`.

The Go version is also bumped to version `1.22.3`.

I've tested this locally with the terminal web UI, and the Loop & Pool flow worked as expected for me. I've also tested the Autopilot flow locally.

### CODE REVIEW

File: terminal.go [hunk 1]
```diff
@@ -138,7 +138,7 @@ var (
 	// to run LiT in remote mode.
 	minimalCompatibleVersion = &verrpc.Version{
 		AppMajor: 0,
-		AppMinor: 17,
```
Comment (ViktorTigerstrom) [inline terminal.go:141]: I may revert this if it turns out that none of our su-servers require v0.18.0

File: README.md [hunk 2]
```diff
@@ -155,6 +156,7 @@ The following table shows the supported combinations:
 
 | LiT               | LND          | Loop         | Faraday       | Pool         | Taproot Assets |
 |-------------------|--------------|--------------|---------------|--------------|----------------|
+| **v0.13.0-alpha** | v0.18.0-beta | v0.28.3-beta | v0.2.13-alpha | v0.6.5-beta  | v0.3.3-alpha   |
```
Comment (ViktorTigerstrom) [inline README.md:159]: Not sure exactly what to set `pool` & `tapd` to here, as those won't be official releases, but rather branches litd will depend on.

File: autopilotserver/mock/server.go [hunk 3]
```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/lightninglabs/lightning-terminal/autopilotserverrpc"
 	"github.com/lightninglabs/lightning-terminal/rules"
 	"github.com/lightninglabs/lightning-terminal/session"
-	"github.com/lightningnetwork/lnd/lntest/node"
+	"github.com/lightningnetwork/lnd/lntest/port"
```
Comment (guggero) [inline autopilotserver/mock/server.go:16]: Unfortunately this commit needs to be squashed with the previous one to make CI happy.

File: README.md (hunk 2 above)
Comment (guggero) [reply README.md:159]: The latest tag versions are okay IMO. So this looks good.

**Review by guggero — APPROVED** [review]
Nice, LGTM once CI is fixed.

**Comment by ViktorTigerstrom** [comment]
Updated to the latest loop release, and also dropped the updates to the minimum required lnd version!

**Review by guggero — APPROVED** [review]
utACK, LGTM :tada:

**Review by ellemouton — APPROVED** [review]
LGTM 🥇

File: app/build/.gitkeep
Comment (jamaljsr) [inline app/build/.gitkeep:1]: Just out of curiosity, why is this file no longer needed?

The [commit](https://github.com/lightninglabs/lightning-terminal/commit/4499e45ac533dfad7c34f558786c7811578c0b5f) where this was originally added states:
> Even though the content of the app/build directory should be ignored by git, we need it to exist in order for the Golang embedded file system compilation to succeed on an empty project. This is required to fix compilation issues when lightning-terminal is used as a module dependency in another project, where the frontend build process can't be triggered.

Was the Golang embedded fs updated so this is no longer needed?

**Review by jamaljsr — APPROVED** [review]
tACK LGTM 🚀

Just did a quick test of Loops in the `litd` UI and connecting from TW.

File: app/build/.gitkeep
Comment (guggero) [reply app/build/.gitkeep:1]: Oh, I missed this in my review... Thanks for spotting! We definitely should keep this file.

**Review by guggero — CHANGES_REQUESTED** [review]

File: app/build/.gitkeep
Comment (ViktorTigerstrom) [reply app/build/.gitkeep:1]: Ah totally my bad! Great catch, thanks!

**Comment by ViktorTigerstrom** [comment]
Addressed @jamaljsr feedback with the latest push!
I also bumped loop to `v0.28.5-beta` that's been released since the last push. Note that this also changes the litd `lnd` dependency to reference the `lnd v0.18.0-beta.1` tag. That tag is referencing the same commit as lnd `v0.18.0-beta` though, so that's not an issue!

**Comment by ellemouton** [comment]
@jamaljsr  - mind giving this another since the update and the rebase?

**Review by jamaljsr — APPROVED** [review]
Reviewed the latest changes. 

tACK LGTM 👌

**Pull request merged**